src/main/java/de/uhd/ifi/se/decision/management/jira/model/PassRule.java

```diff
@@ -0,0 +1,50 @@
+package de.uhd.ifi.se.decision.management.jira.model;
+
+import de.uhd.ifi.se.decision.management.jira.model.cia.IgnoreArgumentsDecisionsPassRule;
+import de.uhd.ifi.se.decision.management.jira.model.cia.IgnoreDecisionIncoming;
+import de.uhd.ifi.se.decision.management.jira.model.cia.PassPredicate;
+import de.uhd.ifi.se.decision.management.jira.model.cia.UndefinedPassRule;
+
+public enum PassRule {

```

Comment:
I think it would be good to have a short comment above this class (and for other classes and methods).